root/components/forms.tt

```diff
@@ -242,36 +242,38 @@
 
 [%- MACRO ipi_bubble BLOCK -%]
     <div class="bubble" id="ipi-bubble">
-      <p>[% l('IPI codes are assigned by CISAC to “interested parties” in musical rights management.
-               Check {ipi_doc|the documentation} for more info.',
-            { ipi_doc => doc_link('IPI') }) %]</p>
+      <p>[% l('{doc|IPI codes} are assigned by CISAC to “interested parties” in musical rights management.',
+              { doc => doc_link('IPI') }) %]</p>
       <p>[% l('If you don’t know the code for this entity, just leave the field blank.') %]</p>
     </div>
 [%- END -%]
 
 [%- MACRO isni_bubble BLOCK -%]
     <div class="bubble" id="isni-bubble">
-      <p>[% l('ISNI codes are an ISO standard used to uniquely identify persons and organizations.
-               Check {isni_doc|the documentation} for more info.',
-            { isni_doc => doc_link('ISNI') }) %]</p>
+      <p>[% l('{doc|ISNI codes} are an ISO standard used to uniquely identify persons and organizations.',
+              { doc => doc_link('ISNI') }) %]</p>
       <p>[% l('If you don’t know the code for this entity, just leave the field blank.') %]</p>
     </div>
 [%- END -%]
 
-[%- MACRO area_bubble BLOCK -%]
-    <div class="bubble" id="area-bubble">
-      <!-- ko with: target() && target().area -->
-        <p data-bind="html: $data.selectionMessage()"></p>
-      <!-- /ko -->
-    </div>
+[%- MACRO area_bubble_selection BLOCK -%]
+    <!-- ko if: target() && target().area && target().area().gid -->
+      <p data-bind="html: target().area().selectionMessage()"></p>
+    <!-- /ko -->
 [%- END -%]
 
 [%- MACRO sort_name_bubble BLOCK -%]
     <div class="bubble" id="sort-name-bubble">
-      <p>[% l('The sort name is an artist name variant used when sorting alphabetically.
-               For usage information check the {sort_name_doc|sort name guidelines}.',
-            { sort_name_doc => doc_link('Style/Artist/Sort_Name') }) %]</p>
-      <p>[% l('“Last Name, First Name” and “Group, The” are common examples of sort names.') %]</p>
+      <p>
+        [% l('The sort name is an artist name variant used when sorting alphabetically.') %]
+      </p>
+      <p>
+        [% l('“Last Name, First Name” and “Group, The” are common examples of sort names.') %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
```

Comment:
I split this into its own paragraph and made it re-use the same string that a bunch of other bubbles use.